core/src/test/java/greencity/service/impl/FavoritePlaceServiceImplTest.java

```diff
@@ -37,6 +38,8 @@
     @Mock
     private FavoritePlaceDtoMapper favoritePlaceDtoMapper;
     @Mock
+    private FavoritePlaceMapper favoritePlaceMapper;
```

Comment:
All these tests use junit4. You need to refactor all of them on junit5.

1) All core annotations are located in the **org.junit.jupiter.api** package in the junit-jupiter-api module.
2) All JUnit Jupiter assertions are static methods in the **org.junit.jupiter.api.Assertions** class.

In JUnit 5, to test **exception** cases (instead @Test(expected = ...class) you should use **org.junit.jupiter.api.Assertions.assertThrows()** method.
simple example:
@Test
void testExpectedException() {
  Assertions.assertThrows(NumberFormatException.class, () -> {
    Integer.parseInt("seven");
  });
}

Reply:
In the scope of current task there was aim to refactor mappers (remove services and make them follow the unified scheme - extending the AbstractConverter) and cover them with respective tests. Migration of all ServiceImpl Tests to JUnit 5 might be added as a task to the backlog to be solved if the next sprint.